common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -734,17 +757,14 @@
             "error"
         ],
         "no-void": [
-            "off"
+            "error"
```

Comment:
I think we should continue to permit void probably.

Reply:
I feel intimidated by the docs: <https://eslint.org/docs/rules/no-void>

I think that anyone who feels confident enough to use void can disable the rule with a comment. It doesn't seem like something most TS devs would need.